scripts/docs/snippetadapter.mjs

```diff
@@ -136,12 +136,31 @@ async function buildSnippets( snippets, paths, constants, imports ) {
 			 * bundled. This will cause issues, because the `tests` directory is not available in the CDN build.
 			 */
 			{
-				name: 'external',
+				name: 'externalize-ckeditor5',
 				setup( build ) {
 					build.onResolve( { filter: /.*/ }, args => ( {
 						external: externals.some( name => name.endsWith( '/' ) ? args.path.startsWith( name ) : args.path === name )
 					} ) );
 				}
+			},
+
+			/**
+			 * Code from "ckeditor5" and "ckeditor5-premium-features" is externalized to avoid bundling it with the snippet
+			 * code. However, if the source file is imported directly, it can be bundled and (in case of commercial features)
+			 * not obfuscated. To prevent this, we throw an error if such an import is detected.
+			 */
+			{
+				name: 'throw-on-source-import',
+				setup( build ) {
+					build.onLoad( { filter: /packages\/(.*)\/src/ }, args => ( {
```

Comment:
@psmyrek, could you check if adding the following snippet

```js
import version from '@ckeditor/ckeditor5-utils/src/version.js';
```

to any snippet, then building them throws an error on Windows?

While I believe all paths are absolute at this stage, I am unsure if they use an OS separator.